### ISSUE TITLE
Overview plugin disables all hotkeys

### ISSUE THREAD

**Issue by Metotron** [codebrainz/overview-plugin#16]
Copy of my message from geany issues:
I have openSUSE 13.2 (Harlequin) (i586)
I make geany from git with this parameters of geany-plugins:
--disable-debugger --disable-devhelp --disable-geanygendoc --disable-geanylatex --disable-geanylipsum --disable-defineformat --disable-geanyctags --disable-geanydoc --disable-geanyinsertnum --disable-geanylua --disable-geanymacro --disable-geanyminiscript --disable-geanysendmail --disable-geanypg --disable-geniuspaste --disable-markdown --disable-multiterm --disable-pohelper --disable-pretty_printer --disable-scope --disable-shiftcolumn --disable-spellcheck --disable-updatechecker --disable-xmlsnippets --disable-geanypy
When I use overview plugin alone or in combination with other, I can not use any hotkeys except overview hotkeys (I use alt+m). Overview may be hidden or visible that time.

**Comment by zmiimz** [comment]
I have the same issue in debian.  Compiled and installed overview directly from github. Nothing changed  since 2015?

**Comment by codebrainz** [comment]
Sorry, I don't personally use this plugin, it was mostly created for a challenge and because people kept asking for it because other editors have it. You could try using it [from Geany-Plugins](https://github.com/geany/geany-plugins) where maintenance happens these days, though I'm not sure whether this issue has been addressed.

I'd be happy if anyone wants to take on fixing this issue or any others, or even to take over being the main maintainer for this plugin.

**Comment by zmiimz** [comment]
ok, then m.b. https://help.github.com/en/articles/disabling-issues

**Comment by codebrainz** [comment]
Good idea. I think I'll just make the repository "Archived" since maintenance is being done in Geany-Plugin repos anyway.
